src/OrchardCore.Modules/OrchardCore.Setup/Views/_Layout.cshtml

```diff
@@ -21,5 +42,16 @@
             @await RenderSectionAsync("Footer", required: false)
         </footer>
     </div>
+    <script>
+        function setLocalizationUrl() {
+            var culturesList = document.getElementById('culturesList');
+            var culture = culturesList.options[culturesList.selectedIndex].value
+            var queryString = window.location.search;
+            var searchParams = new URLSearchParams(queryString);
+
+            searchParams.set("ui-culture", culture);
+            window.location = window.location.href.split('?')[0] + '?' + searchParams.toString();
+        }
```

Comment:
the tenant url (if present) might be better retrieved from the `HttpContext.PathBase`